test/server/cards/02_SHD/events/MomentOfGlory.spec.js

```diff
@@ -0,0 +1,30 @@
+describe('Covert Strength', function () {
+    integration(function () {
+        describe('Covert Strength\'s ability', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['moment-of-glory'],
+                        groundArena: [{ card: 'pyke-sentinel', damage: 1 }],
+                        leader: { card: 'sabine-wren#galvanized-revolutionary', deployed: true, damage: 4 }
+                    },
+                    player2: {
+                        groundArena: ['wampa'],
+                        spaceArena: ['imperial-interceptor']
+                    }
+                });
+            });
+
+            it('can buff a unit', function () {
+                this.player1.clickCard(this.momentOfGlory);
+                expect(this.player1).toBeAbleToSelectExactly([this.pykeSentinel, this.sabineWren, this.wampa, this.imperialInterceptor]);
+
+                this.player1.clickCard(this.pykeSentinel);
```

Comment:
This is another case where we don't want to just check a property, we prefer to check a real game effect. For buffs like this I will typically pick an enemy unit that would normally kill the attacker but since it has the buff it does not die.

So if Pyke Sentinel is normally 2/3 and is now 6/7 I'll pick something with (3 < power < 7) and hp > 6. So like AT-ST would be a nice simple one.